aiida/tools/importexport/dbimport/backends/django/__init__.py

```diff
@@ -471,9 +472,9 @@ def import_data_dj(
                 if 'mtime' in [field.name for field in model._meta.local_fields]:
                     with models.suppress_auto_now([(model, ['mtime'])]):
                         # Store them all in once; however, the PK are not set in this way...
-                        model.objects.bulk_create(objects_to_create)
+                        model.objects.bulk_create(objects_to_create, batch_size=get_config_option('db.batch_size'))
```

Comment:
probably better to just fetch the value once and assign to local variable that is to be reused

Reply:
hm yes... I was thinking there might be cases where you are creating a new profile, where you don't really want to use the setting of the current profile, so I thought it might be safer to start with a global option.
Anyhow, I think making it not only global is fine

Reply:
I think you meant this to be a reply to the other comment ;) Anyway, that is not really what the `global_only` means. When an option is `global_only` it means it can only be set instance-wide and not per profile. Currently this only applies to the `user_*` options used for making repeated profile creation easier, so there it doesn't make sense as profile specific. The resolution order of configuration settings is `profile` -> `global` -> `default`. So if an existing profile defines a specific `db.batch_size` for itself, a new profile would still get the default, because there is nothing set globally.